gcp/modules/couchdb/main.tf

```diff
@@ -133,7 +140,9 @@ resource "null_resource" "couchdb_destroy_pvcs" {
 
     command = <<EOF
       for PVC in $(kubectl get pvc --namespace ${var.release_namespace} -o json | jq --raw-output '.items[] | select(.metadata.name | startswith("database-storage-couchdb")) | .metadata.name'); do
-        kubectl --namespace ${var.release_namespace} delete --ignore-not-found pvc $PVC
+        if [ "${var.execute_destroy_pvcs}" == "true" ]; then
```

Comment:
why don't you put the `for` loop inside the if condition, in order to avoid unnecessary `kubectl` commands?

Reply:
Nice catch!